contrib/depend/install.sh

```diff
@@ -139,4 +151,6 @@ elif [ $1 == "xtensor-python" ]; then
 elif [ $1 == "everything" ]; then
   pybind11
   modmesh
+elif [ $1 == "ubuntu_1804" ]; then
```

Comment:
It is a bad idea to mix installation scripts with a package manager and with source building.